src/mawpy/utilities/cluster.py

```diff
@@ -0,0 +1,47 @@
+"""
+data structure of a cluster
+"""
+
+import numpy as np
+
+from mawpy.distance import distance
+
+
+class Cluster:
+    def __init__(self):
+        self.pList = []
+        self.center = [0, 0]
+        self.radius = 0
+
+    def add_point(self, point):
```

Comment:
what is the type of point? I thought it would already be Tuple[float]?

Reply:
yes, updated